### DISCUSSION TITLE
ASDB Exchange forks / clones

### DISCUSSION THREAD

**Discussion by RType23** [ketilmo/balena-ads-b#76] [General]
Hi - with the recent news of ADSBx and the commercial deal fracturing the community I'd like to support the other ADSBx clones that have popped up over the past 2 weeks.  I've very little git experience however.  I assume I need to clone the ADSBx section and populate to feed the new sites, and mirror or reuse the variables, and something to start each service.  I'm using a Pi 4 4GB so have plenty of resources to use.

Any pointers you could advise?

Many thanks for the excellent project!
Rich

**Comment by ketilmo** [comment]
Hi @RType23,

Thanks for reaching out. It's great that you want to contribute to the project. What ADSBX clones are you looking into? They may be relatively straightforward to implement if they are similar to other alternatives.

**Comment by RType23** [comment]
Hi @ketilmo - thanks for the reply. These are the community run clones of adsbx:

https://globe.adsb.fi/
https://globe.adsb.one/
https://globe.adsb.lol/
https://globe.theairtraffic.com/  (Chris's message [here](https%3A%2F%2Fgithub.com%2Fketilmo%2Fbalena-ads-b%2Fissues%2F77&data=05%7C01%7C%7C02c67bf928f84dc0e03a08db1bcd13c2%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C638134339832196954%7CUnknown%7CTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7C3000%7C%7C%7C&sdata=FBP872hDu0MXntQxapT%2FPkNzWo%2F2fPPHoTvp%2Bgfcm%2Fg%3D&reserved=0)

Thanks!

**Comment by jason0x21** [comment]
I'm interested in this as well, and would like to help.

**Comment by ketilmo** [comment]
Hi everyone,

I'm sorry for not getting back to you sooner. Things have been a bit hectic over the past few months. The number of ADSB Exchange services seems to have exploded over the last couple of months. Given that they are compatible, feeding several of them from the same service might be possible. Unfortunately, I don't have much time to look into this in-depth.
